packages/forms/src/styles.ts

```diff
@@ -6,8 +6,8 @@ const fieldStateStyles: Record<FieldState, Record<string, string>> = {
     "data-h2-border-color": "base(gray) base:focus-visible(focus)",
   },
   invalid: {
-    "data-h2-border-color": "base(error.darker) base:focus-visible(focus)",
     "data-h2-background-color": "base(error.lightest)",
+    "data-h2-border-color": "base(error.darker) base:focus-visible(focus)",
```

Comment:
Are you like me and try to alpha sort properties 👀 😛

Reply:
😬 I always get judged for this, but yes. It makes finding a property/selector/attribute so much easier.